app/brave_main_delegate_browsertest.cc

```diff
@@ -19,7 +19,7 @@
 #include "components/safe_browsing/core/features.h"
 #include "content/public/browser/render_view_host.h"
 #include "content/public/common/content_features.h"
-#include "content/public/common/web_preferences.h"
+#include "third_party/blink/public/common/web_preferences/web_preferences.h"
```

Comment:
there's already a dep for this?

Reply:
`third_party/blink/public/common/features.h`, which is part of the same GN target than this include, is also included down below in this same file, so this should be fine.